src/proto/misc1.pro

```diff
@@ -55,5 +55,9 @@ void restore_v_event(dict_T *v_event, save_v_event_T *sve);
 void may_trigger_modechanged(void);
 int vim_append_digit_int(int *value, int digit);
 int vim_append_digit_long(long *value, int digit);
+#ifdef HAVE_NO_LONG_LONG
+int trim_to_int(long x);
+#else
 int trim_to_int(long long x);
+#endif
```

Comment:
As I said above, *.pro files are generated files. This change will be overwritten next time.